components/segment-bridge/base/role.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
This gives us cluster-wide permissions, while we only need permissions for a particular namespace.

Reply:
fixed